python/mlcroissant/mlcroissant/_src/core/constants.py

```diff
@@ -161,6 +161,7 @@ class EncodingFormat:
     """
 
     CSV = "text/csv"
+    DIR = "application/x-dir"
```

Comment:
What's the difference with https://github.com/mlcommons/croissant/blob/e532c4c17d97ee660f40f2c21ac12b66ca0e990f/datasets/1.0/recipes/read_from_directory.json#L51-L59?